graph/src/test/java/smile/graph/AdjacencyListTest.java

```diff
@@ -349,8 +349,6 @@ public void testDfs() {
     @Test
     public void testDfs2() {
         System.out.println("dfs connected component");
```

Comment:
Please keep these variables. I may miss some test on these results. I should add more tests, not remove them. Thanks!